packages/lib-classifier/src/components/Classifier/components/Feedback/FeedbackModal.spec.js

```diff
@@ -16,6 +22,28 @@ describe('FeedbackModal', function () {
     expect(wrapper.html()).to.be.null
   })
 
+  it('should show FeedbackViewer if hideSubjectViewer false and applicableRules', function () {
```

Comment:
How about we add sub describe blocks for `when subject viewer is hidden` and for `when subject viewer is shown`? The it statements are a little hard to parse for this set.